c7n/resources/glue.py

```diff
@@ -412,3 +412,66 @@ def process(self, resources):
                 client.delete_workflow(Name=r['Name'])
             except client.exceptions.EntityNotFoundException:
                 continue
+
+
+@resources.register('glue-catalog')
+class GlueDataCatalog(QueryResourceManager):
```

Comment:
catalog is a synthetic account resource, see rest api or account as examples, its not query resource manager derived

Reply:
Thanks for the reference! Addressed it